mozilla_aws_cli/login.py

```diff
@@ -346,21 +327,14 @@ def get_role_map(self):
 
         if self.role_map is None:
             self.exit("Unable to retrieve role map. Shutting down.")
+            return False
 
         logger.debug(
             'Roles and aliases are {}'.format(self.role_map))
+        return self.role_map
 
 
     def exchange_token_for_credentials(self):
```

Comment:
I think this can also be called multiple times, is that okay with calling into `sts_conn.get_credentials()`?

Reply:
`get_credentials` is actually only called in one place in the code. I've thought about moving it into `login` as that's the only thing that calls it.